### ISSUE TITLE
Keep getting error "You cant mount here" on some mounts in zereth mortis

### ISSUE THREAD

**Issue by zanedasavage** [xod-wow/LiteMount#95]
What happens?

Most of the mounts I've tried on my alts work (at least the ones I've tried). However 2 that I can get to work are Ban-Lu, Grandmaster's Companion and Hogrus, Swine of Good Fortune.

What did you expect to happen instead?

I'm not sure if I understand the question...but I expected it to fly...lol

Did it work in a previous version of LiteMount? If so, what was the last version that worked?

First time using LiteMount

Had to remove "option db" because it wouldn't let me submit...said it was too long.

--- General ---

date: Tue Jul  5 05:04:52 2022
version: 9.2.15
locale: enUS
current profile: IRONFIST

--- Player ---

name: Ìronfìst-Dalaran
class: MONK
level: 60
race: Pandaren
faction: Horde
spec: 3 269 Windwalker

--- Location ---

map: Zereth Mortis (1970)
mapPath: Zereth Mortis (1970) -> The Shadowlands (1550) -> Cosmic (946)
instance: Zereth Mortis (2374)
zoneText: Zereth Mortis
subZoneText: 
IsFlyableArea(): true

--- Debugging Output ---

 - checking profile: IRONFIST
VersionUpgrade: 8
 - checking profile: IRONFIST
VersionUpgrade: 8
 - checking profile: IRONFIST
Initializing LiteMount v9.2.15, debugging enabled.
PostClick handler called on LM_B1
 - setting action to in-combat action
PostClick handler called on LM_B2
 - setting action to in-combat action
PostClick handler called on LM_B3
 - setting action to in-combat action
PostClick handler called on LM_B4
 - setting action to in-combat action
Got refresh event BAG_UPDATE_DELAYED
PreClick handler called on LM_B1
Refreshing status of all mounts.
Dispatching rule LeaveVehicle
Dispatching rule Dismount [nofalling]
Dispatching rule CopyTargetsMount
Dispatching rule ApplyRules
 - checking 3 rules for button 1
Dispatching rule LimitSet [flyable] Ban-Lu, Grandmaster's Companion
 - new filter: =Grandmaster's Companion
 - no rules matched
Dispatching rule SmartMount
 - filters: =Grandmaster's Companion
 - filtered list contains 0 mounts
Dispatching flow control action IF [falling]
 - IF test is false
Dispatching flow control action END
Dispatching rule Macro
 - setting action to can't mount now
PostClick handler called on LM_B1

**Comment by mbattersby** [comment]
Yep, there's definitely a bug. Looks like it affects mounts with commas in the name.

**Comment by mbattersby** [comment]
I'm about to push a fix for this which will close your ticket. You'll need to recreate your Ban-lu rule for it to work. Even though it looks right it's broken behind the scenes.

**Comment by mbattersby** [comment]
Fix will be in LiteMount 9.2.16.
